src/components/Dropdown/dropdown.module.css

```diff
@@ -62,10 +62,23 @@
     border-top-left-radius: var(--shape-border-radius-lg, 8px);
     border-top-right-radius: var(--shape-border-radius-lg, 8px);
   }
+
+  :global(.mia-platform-tag) {
+    margin-inline-end: unset;
+  }
 }
 
-.labelContainer {
+.labelOuterContainer {
```

Comment:
since we already have the "label container" concept we could name this one differently; could this be something that recalls the "item row" concept?

Reply:
makes sense to me